src/Middleware/CORS/test/FunctionalTests/package.json

```diff
@@ -3,7 +3,7 @@
   "devDependencies": {
     "jest": "^23.6.0",
     "merge": "^1.2.1",
-    "puppeteer": "^1.12.2"
+    "puppeteer": "^1.13.0"
```

Comment:
Undo?

Reply:
The build keeps updating this file. I've decided to stop fighting it at this point